docs/contributing/running-and-writing-acceptance-tests.md

```diff
@@ -496,6 +496,36 @@ resource "aws_example_thing" "test" {
 }
 ```
 
+#### Service Availability Test
```

Comment:
I'd recommend naming this new section `Service Availability PreCheck` and it needs to be added to the page table of contents. These `PreCheck` are not implemented globally at the moment (or included with the examples on the rest of the page) so may be worth sticking outside the main flow of the Resource Acceptance Testing for now.

Reply:
Changed label, and added to the table of contents. I think it would be odd to move it outside the main flow, even if it isn't widely implemented. So I added a note attempting to explain it.